src/components/Sidebar.tsx

```diff
@@ -58,6 +60,35 @@ export interface SidebarItem {
   roles?: Role | Role[],
 }
 
+export interface ManageMenuItem {
```

Comment:
`DropdownMenuItem`. Also remove `export`. Can remove `export` from `SidebarItem` in the same PR.